notebooks/milestone1.ipynb

```diff
@@ -1237,18 +1229,10 @@
     "\n",
     "1. One of the challenges we had with Q5 was the long running time. For example, initially tried to use `.info()` like what we did in other courses to obtain the `dtype` for each column. However, it took a long time and output additional information we were not particularly interested in, such as the number of the non-null values in each column. Alternatively, we used `.dtypes` to get the data type of columns and it returned the results quickly. \n",
     "\n",
-    "2. For EDA in R, we were not able to create a very comprehensive EDA with sophisticated plots because of the size of the data. We managed to plot a simple line plot to show the trend of rain over years. Due to the large data size, we ended up aggregating it with means of rainfall by years instead of plotting the original rainfall data. The aggregation process took about 10 minutes for Chaoran and it seemed to be acceptable, but the kernels on Junrong's and Amelia's laptops died after running the cell for the plot for a long time. We could not overcome this challenge due to our limited computational power. If we could load this notebook to AWS SageMaker, we all should be able to run the plot.    \n",
+    "2. For EDA in R, we were not able to create sophisticated plots when performing exploratory data analysis due to the large data size. We managed to plot a simple line plot to show the trend of rain over years. We aggregated the data with means of rainfall by years instead of plotting the original rainfall data. If we could load this notebook to a cloud, such as AWS SageMaker, we might be able to generate more sophisticated plots. \n",
     "\n",
     "3. We had quite some discussions over which file format to choose between`feather` or `parquet` because one had the advantage on loading time and the other had the advantage on memory usage. We resolved this by running a few experiments and picked `parquet` because obvious smaller data size but only a little bit slower than `feather`. "
```

Comment:
```suggestion
    "3. We had quite some discussions over which file format to choose between`feather` or `parquet` because one had the advantage on loading time and the other had the advantage on memory usage. We resolved this by running a few experiments and picked `parquet` because the `parquet` file had a significantly smaller size but the time it took to load in R was only slightly longer than what the `feather` file took.  
```